x-pack/platform/plugins/shared/fleet/server/services/package_policies/package_policies_aggregation.ts

```diff
@@ -18,6 +18,7 @@ export async function getPackagePoliciesCountByPackageName(soClient: SavedObject
   >({
     type: savedObjectType,
     perPage: 0,
+    filter: `${savedObjectType}.attributes.latest_revision:true`,
```

Comment:
Need to filter out previous revisions from the count in Installed integrations table - package policies count